app/v2/controllers/requestParsers/validators/validations/JsonValidation.scala

```diff
@@ -16,18 +16,48 @@
 
 package v2.controllers.requestParsers.validators.validations
 
-import play.api.libs.json.{Reads, _}
-import v2.models.errors.MtdError
+import play.api.Logger
+import play.api.libs.json._
+import v2.models.errors.{MtdError, RuleIncorrectOrEmptyBodyError}
 
 /**
-  * Utilities to assist using validations where the value to validate comes from a JSON element
-  */
+ * Utilities to assist using validations where the value to validate comes from a JSON element
+ */
 object JsonValidation {
```

Comment:
This file has 0% coverage.

Reply:
This file is unused anywhere in the application, thanks for finding